src/core_plugins/tagcloud/public/tag_cloud_controller.js

```diff
@@ -19,6 +19,7 @@ module.controller('KbnTagCloudController', function ($scope, $element, Private,
     const aggs = $scope.vis.getAggConfig().getResponseAggs();
     const aggConfigResult = new AggConfigResult(aggs[0], false, event, event);
     clickHandler({ point: { aggConfigResult: aggConfigResult } });
+    $scope.$apply();
```

Comment:
is this really needed ?  are you sure it will never run in the middle of digest cycle ?

Reply:
Not sure about never running in the middle of the digest cycle but without it - the filter is not created until you do soemthing else that runs a digest cycle

Reply:
what about using the new  `vis.type.API.click` ?